frontend/components/Blocks/CardsBlock/Card.tsx

```diff
@@ -0,0 +1,44 @@
+import RawHtml from "../../RawHtml/RawHtml";
+import Image from "next/image";
+
+type CardItem = {
+  title: string;
+  image_selector: {
+    id: number;
+    title: string;
+    img: {
+      src: string;
+      width: number;
+      height: number;
+      alt: string;
+    };
+  };
+  text: string;
+  card_background: string;
+};
+
+export default function Card({ cardItem, key }) {
```

Comment:
This type isn't used inside the Card component. I'm guessing it was meant to be?

```suggestion
type CardItem = {
  title: string;
  imageSelector: {
    id: number;
    title: string;
    img: {
      src: string;
      width: number;
      height: number;
      alt: string;
    };
  };
  text: string;
  cardBackground: string;
};

type Props = {
  cardItem: CardItem;
  key: string;
}

export default function Card({ cardItem, key }: Props) {
```

Reply:
Thanks, you're right, I've changed this in the code.